docs/cog_guides/mutes.rst

```diff
@@ -0,0 +1,409 @@
+
+.. tip:: Alias: ``channelmute``
+
+**Description**
+
+Mute a user in the current text channel.
+
+Examples:
+
+* ``[p]mutechannel @member1 @member2 spam 5 hours``
+* ``[p]mutechannel @member1 3 days``
+
+**Arguments**
+
+* ``<users...>``: A space separated list of usernames, ID's, or mentions.
+* ``[time_and_reason]``: The time and reason. If no time is provided, the mute will use the default set time or indefinite if this hasn't been configured.
+
+.. _mutes-command-muteset:
+
+^^^^^^^
+muteset
+^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]muteset 
+
+**Description**
+
+Mute settings.
+
+.. _mutes-command-muteset-defaulttime:
+
+"""""""""""""""""""
+muteset defaulttime
+"""""""""""""""""""
+
+.. note:: |mod-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]muteset defaulttime [time]
+
+.. tip:: Alias: ``muteset time``
+
+**Description**
+
+Set the default mute time for the mute command.
+
+If no time interval is provided this will be cleared.
+
+**Arguments**
+
+* ``[time]``: The length of time for a default mute.
+
+.. _mutes-command-muteset-forcerole:
+
+"""""""""""""""""
+muteset forcerole
+"""""""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]muteset forcerole <true_or_false>
+
+**Description**
+
+Whether or not to force role only mutes on the bot.
+
+**Arguments**
+
+* ``<true_or_false>``: Whether or not to enable/disable this setting, most provide ``true`` or ``false``.
+
+.. _mutes-command-muteset-makerole:
+
+""""""""""""""""
+muteset makerole
+""""""""""""""""
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]muteset makerole <name>
+
+**Description**
+
+Create a Muted role.
+
+This will create a role and apply overwrites to all available channels
+to more easily setup muting a user.
+
+If you already have a muted role created on the server use
+``[p]muteset role ROLE_NAME_HERE``
+
+**Arguments**
+
+* ``<name>``: The name of the muted role to create.
+
+.. _mutes-command-muteset-notification:
+
+""""""""""""""""""""
+muteset notification
+""""""""""""""""""""
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]muteset notification [channel]
+
+**Description**
+
+Set the notification channel for automatic unmute issues.
+
+If no channel is provided this will be cleared and notifications
+about issues when unmuting users will not be sent anywhere.
+
+**Arguments**
+
+* ``[channel]``: The channel to receive unmute issue updates. |channel-input|
+
+.. _mutes-command-muteset-role:
+
+""""""""""""
+muteset role
+""""""""""""
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]muteset role [role]
+
+**Description**
+
+Sets the role to be applied when muting a user.
+
+If no role is setup the bot will attempt to mute a user by setting
+channel overwrites in all channels to prevent the user from sending messages.
+
+.. Note:: If no role is setup a user may be able to leave the server
+
+and rejoin no longer being muted.
```

Comment:
![image](https://user-images.githubusercontent.com/34169552/117520144-c91b2a80-af74-11eb-8f60-e2b8f203397e.png)
The syntax on this is broken